neard/src/runtime.rs

```diff
@@ -879,8 +879,8 @@ impl RuntimeAdapter for NightshadeRuntime {
         ) {
             Ok(result) => Ok(result),
             Err(e) => match e.kind() {
-                ErrorKind::StorageError => {
-                    panic!("Storage error. Corrupted db or invalid state.");
+                ErrorKind::StorageError(_) => {
+                    panic!("{}", e);
```

Comment:
Can we keep the `Storage error: {}` around? The underlying error without the enclosing context can be confusing.

Reply:
It will display `Storage error` because of https://github.com/nearprotocol/nearcore/blob/4e2df72f0a9dddba129291ad5928dc730cdc48a5/chain/chain/src/error.rs#L165. Unless you want to see `Storage error: Storage error: <actual error>`.